src/core/client/stream/tabs/Comments/Stream/AllCommentsTab/CommentEnteredSubscription.tsx

```diff
@@ -116,6 +176,12 @@ function insertReply(
     return;
   }
 
+  if (depth >= 4 && flattenLastReply) {
```

Comment:
Agree the `4` should be extracted into a well named constant.

Reply:
Agreed, did this in previous fixes requested by @tessalt